etc/releasing.md

```diff
@@ -431,6 +437,7 @@ follows:
   distribution tarball.
 - If the script succeeded at creating the distribution tarball, pass it directly
   with `--dist-file ./build/mongo-cxx-driver-rX.Y.Z.tar.gz`.
+- If the only open JIRA ticket is the ticket tracking the release itself, pass `--allow-open-issues`.
```

Comment:
```suggestion
```

I expect this is not needed. An automation creates the "Release x.y.z" ticket when a fixVersion is added. The created ticket gets a `release` label (example CXX-3104). Tickets with the `release` label are [filtered out](https://github.com/mongodb/mongo-cxx-driver/blob/2a118824477fc5f21e01d8c873c5cab27b617b04/etc/make_release.py#L506).

I expect the "Release 4.0.0" ticket was created manually and did not get the label.